bbc/simorgh
# WSTEAM1-1058: Update tick SVG to SUCCESS_CORE colour

Resolves JIRA [WSTEAM1-1058](https://jira.dev.bbc.co.uk/browse/WSTEAM1-1058) - A11y P1 - SVG Colour contrast too low

Overall changes
======
Applies updated SUCCESS_CORE colour to SVG tick

Code changes
======
- Updates/ add SUCCESS_CORE to palette in the Theme provider
- Applies colour to SVG tick via CSS.

Note this means the tick appears black without CSS. This the same as we do for the Live Label pulse and the brand SVG, so I think this is ok.

Testing
======
1. View Success state via storybook permalink
2. Check colour matches design
3. Check colour passes colour contrast check

Helpful Links
======
_Add Links to useful resources related to this PR if applicable._

[Coding Standards](https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/README.md)

[Repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
